containers/bastion/cockpit-auth-ssh-key

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+import time
+import base64
+import subprocess
+
+COCKPIT_SSH_COMMAND = "/usr/libexec/cockpit-ssh"
+
+def usage():
+    sys.stderr.write("usage {} [user@]host[:port]\n".format(sys.argv[0]))
+    sys.exit(os.EX_USAGE)
+
+def send_auth_command(challenge, response):
```

Comment:
Could you add a comment  near the top of the file with a link to protocol.md describing what we're w riting on stdout and reading from stdin?

Reply:
Added